build-system/tasks/ava.js

```diff
@@ -15,20 +15,24 @@
  */
 'use strict';
 
-const ava = require('gulp-ava');
-const gulp = require('gulp-help')(require('gulp'));
+const gulp = require('gulp');
+const gulpAva = require('gulp-ava');
 const {isTravisBuild} = require('../travis');
 
 /**
  * Runs ava tests.
  */
-function runAvaTests() {
+function ava() {
```

Comment:
A bit strange to mirroring the name 'ava', not ideal but you could export this function as 'ava', but continue to call it 'runAvaTests' internally.

`module.exports = { ava: runAvaTests }` I believe would do this.

Not sure it's any better though.